python/tests/unit/arcticdb/version_store/test_normalization.py

```diff
@@ -127,10 +127,10 @@ def test_multiindex_with_tz(tz):
     df, norm_meta = norm.normalize(d)
     fd = FrameData.from_npd_df(df)
     denorm = norm.denormalize(fd, norm_meta)
-    if pd.__version__.startswith("1"):
-        assert_equal(d.index.to_numpy(), denorm.index.to_numpy())
-    else:
+    if pd.__version__.startswith("0"):
         assert_equal(d.index.get_values(), denorm.index.get_values())
+    else:
+        assert_equal(d.index.to_numpy(), denorm.index.to_numpy())
```

Comment:
I wonder if `pandas.testing.assert_index_equal` existed for the `0.x` releases?

Reply:
It came in for v.0.23, I think we're still supporting v0.22 😢 
https://pandas.pydata.org/pandas-docs/version/1.0/whatsnew/v0.23.1.html?highlight=assert_index_equal